src/variorum/config_architecture.c

```diff
@@ -108,6 +115,7 @@ int variorum_detect_arch(void)
     if (g_platform.intel_arch == NULL &&
         g_platform.amd_arch   == NULL &&
         g_platform.ibm_arch   == NULL &&
+        g_platform.nvidia_arch   == NULL &&
```

Comment:
Can we align all items in this if block (`== NULL`) with `g_platform.nvidia_arch   == NULL &&`?

Reply:
Fixed the alignment.